app/(pages)/professional/[id]/page.tsx

```
@@ -0,0 +1,455 @@
+                                            setReplyText("")
+                                          }}
+                                        >
+                                          Cancel
+                                        </Button>
+                                      </div>
+                                    </div>
+                                  ) : (
+                                    <button
+                                      className="mt-2 text-xs text-indigo-600 hover:text-indigo-800 font-medium"
+                                      onClick={() => setReplyingTo(review._id)}
+                                    >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)